ic-os/defs.bzl

```diff
@@ -553,17 +553,19 @@ EOF
         ],
         outs = ["launch_local_vm_script"],
         cmd = """
-        IMAGE="$(location :disk-img.tar)"
+        IMAGE="$$PWD/$(location :disk-img.tar)"
         cat <<EOF > $@
 #!/usr/bin/env bash
 set -euo pipefail
 cd "\\$$BUILD_WORKSPACE_DIRECTORY"
```

Comment:
nit: This `cd` is no longer necessary with the full path.

Reply:
Thanks, but the next PR on top of this one already fixes this issue.

Reply:
In general I think we should not merge any code to master that is missing fixes, but this case is just a nit.